src/containers/Teams/TeamOverview/index.tsx

```diff
@@ -1,62 +1,47 @@
 import {A, DocList} from 'components';
-import {TeamName} from 'types/teams';
-import {getTeamData} from 'utils/data';
+import {CoreTeam} from 'types/teams';
 
 import './TeamOverview.scss';
 
 interface ComponentProps {
-  teamFilter: TeamName;
+  team: CoreTeam;
 }
 
-const TeamOverview: FC<ComponentProps> = ({teamFilter}) => {
-  const {description, platforms, responsibilities} = getTeamData(teamFilter);
-
+const TeamOverview: FC<ComponentProps> = ({team}) => {
   const renderTeamDescription = useCallback((): ReactNode => {
     return (
       <>
         <h4 className="TeamOverview__sub-heading"> About the team </h4>
-        <p className="TeamOverview__description">{description}</p>
-        {platforms.map(({name, label, link}) => (
-          <div className="TeamOverview__social" key={`${name}-${label}`}>
-            <Icon
-              className="TeamOverview__social-icon"
-              icon={name === 'github' ? IconType.github : IconType.discord}
-              size={18}
-            />
-            {name === 'github' ? (
-              <A className="TeamOverview__social-title" href={link}>
-                {label}
-              </A>
-            ) : (
-              <p className="TeamOverview__social-title TeamOverview__social-title--sail-gray"> #{label} </p>
-            )}
-          </div>
-        ))}
+        <p className="TeamOverview__description">{team.about}</p>
+        <div className="TeamOverview__social">
+          <Icon className="TeamOverview__social-icon" icon={IconType.github} size={18} />
+          <A className="TeamOverview__social-title" href={team.github}>
+            {team.github}
+          </A>
+        </div>
+        <div className="TeamOverview__social">
+          <Icon className="TeamOverview__social-icon" icon={IconType.discord} size={18} />
+          <p className="TeamOverview__social-title TeamOverview__social-title--sail-gray"> #{team.slack} </p>
+        </div>
       </>
     );
-  }, [description, platforms]);
+  }, [team]);
 
   const renderTeamResponsibilities = useCallback((): ReactNode => {
+    // note: responsibilities is stored as a string in the BE, currently dividing into each point
+    // based on newline. May need to ask BE to store it as an array of strings instead.
+    const responsibilities = team.responsibilities.split('\n');
     return (
       <>
         <h4 className="TeamOverview__sub-heading"> Role and Responsibilities </h4>
         <DocList className="TeamOverview__responsibilities-item" variant="ul">
-          {responsibilities.map(({item, subitems}, i) => (
-            <Fragment key={i}>
-              <li> {item} </li>
-              <ul>
-                {subitems.map((subitem, index) => (
-                  <li key={index}>{subitem}</li>
-                ))}
-              </ul>
-            </Fragment>
-          ))}
+          {responsibilities && responsibilities.map((responsibility) => <li>{responsibility}</li>)}
```

Comment:
@ThaDeveloper also, the current `Team` model has the `responsibilities` as a string field, is it possible to change it to an array of strings to accommodate teams with more than one responsibilities? i.e. 

<img width="1078" alt="Screen Shot 2021-04-10 at 6 22 40 PM" src="https://user-images.githubusercontent.com/32864116/114266602-bd961b80-9a29-11eb-8787-06becdf23a51.png">

Reply:
hey @jamessspanggg i'm not sure if he has his notifications on, could you create `issues` on the `website-api` for these issues you are bringing up?

Reply:
sure will do that